frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -52,11 +54,14 @@ const Uploads = () => {
         return;
       }
 
-      if (currentFile.size > PAYLOAD_MAX_BYTES) {
-        const maxKBytes = (PAYLOAD_MAX_BYTES / 1024).toLocaleString("en-US", {
-          maximumFractionDigits: 2,
-          minimumFractionDigits: 2,
-        });
+      if (currentFile.size > MAX_CSV_UPLOAD_BYTES) {
+        const maxKBytes = (MAX_CSV_UPLOAD_BYTES / 1024).toLocaleString(
```

Comment:
if you upload a file over 50MB then the file does not clear unlike other errrors 🤔 From this diff, I'm not seeing why that's happening.
<img width="705" alt="Screen Shot 2023-01-13 at 09 11 51" src="https://user-images.githubusercontent.com/20211771/212355876-efb49b3e-9190-475b-b38b-09751bb9dc46.png">

Reply:
Resolved offline! We needed to move up the `await file.text()` call to force the file to load, which we _think_ triggered an onChange for the `input` element.